Content.Shared/GameObjects/Components/Sound/SharedSoundComponent.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using System.Collections.Generic;
+using Content.Shared.GameObjects;
+using SS14.Shared.Audio;
+using SS14.Shared.GameObjects;
+using SS14.Shared.Interfaces.GameObjects;
+using SS14.Shared.IoC;
+using SS14.Shared.Map;
+using SS14.Shared.Serialization;
+using SS14.Shared.Timers;
+
+namespace Content.Shared.GameObjects.Components.Sound
+{
+    public class SharedSoundComponent : Component
+    {
+        public override string Name => "Sound";
+        public override uint? NetID => ContentNetIDs.SOUND;
+
+        public override bool NetworkSynchronizeExistence => true;
+    }
+
+    [NetSerializable, Serializable]
+    public class SoundScheduleMessage : ComponentMessage
+    {
+        public SoundSchedule Schedule;
+        public SoundScheduleMessage()
+        {
+            Directed = true;
+        }
+    }
+
+    [NetSerializable, Serializable]
+    public class StopSoundScheduleMessage : ComponentMessage
+    {
+        public string Filename;
+        public StopSoundScheduleMessage()
+        {
+            Directed = true;
+        }
+    }
+
+    [NetSerializable, Serializable]
+    public class StopAllSoundsMessage : ComponentMessage
+    {
+        public StopAllSoundsMessage()
+        {
+            Directed = true;
+        }
+    }
+
+    public enum SoundType
+    {
+        /// <summary>
+        /// Sound follows the entity.
+        /// </summary>
+        Normal,
+        /// <summary>
+        /// Sound is played without position.
+        /// </summary>
+        Global,
+        /// <summary>
+        /// Sound is static in a given position.
+        /// </summary>
+        Positional,
+    }
+
+    [Serializable, NetSerializable]
+    public class SoundSchedule
```

Comment:
I don't know about others (@PJB3005, @Acruid, @clusterfack) but I'd prefer `ScheduledSound` to `SoundSchedule`
as this class represents one sound that has been scheduled, not a whole schedule of sounds.

Reply:
Hmm, you're right. ScheduledSound sounds better, and I think it's easier to understand. I'll change it